src/package-linker.js

```diff
@@ -296,7 +296,17 @@ export default class PackageLinker {
     // remove all extraneous files that weren't in the tree
     for (const loc of possibleExtraneous) {
       this.reporter.verbose(this.reporter.lang('verboseFileRemoveExtraneous', loc));
-      await fs.unlink(loc);
+
```

Comment:
Can we just ignore case in fs.copyBulk instead?

fs.js: 153
```
  for (const loc of possibleExtraneous) {
    if (files.has(loc)) {
      possibleExtraneous.delete(loc);
    }
  }
```

Just call `toLocaleLowerCase()` when dealing with files Set.

Same thing should be applied to buildActionsForHardlink function